sqlalchemy_imageattach/file.py

```diff
@@ -4,13 +4,16 @@
 The file-like types which wraps/proxies an other file objects.
 
 """
-import collections.abc
+try:
+    from collections.abc import Iterator
+except ImportError:
+    from collections import Iterator
```

Comment:
It'd better to simply `import collections` since it has maintained backward incompatible names as well.